Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM httpd:2.4
+COPY ./httpd/my-httpd.conf /usr/local/apache2/conf/httpd.conf
+COPY ./httpd/.htaccess ./web-build /usr/local/apache2/htdocs/
```

Comment:
Ist das getestet? Sieht für mich nämlich so aus, als würdest du den Ordner 'web-build' in htdocs kopieren, aber eigentlich müsste das doch der Ordnerinhalt sein stattdessen.